packages/react/src/SegmentedControl/SegmentedControl.docs.json

```diff
@@ -111,6 +111,12 @@
           "defaultValue": "",
           "description": "Whether the segment is selected. This is used for uncontrolled SegmentedControls to pick one SegmentedControlButton that is selected on the initial render."
         },
+        {
+          "name": "counter",
```

Comment:
Random question for you @langermank, do you know why for `Button` we call this `count`. but for `UnderlineNav.Item` it's `counter`? 🤔

Reply:
I can update!